extensions/smallrye-health/deployment/src/main/java/io/quarkus/smallrye/health/deployment/SmallRyeHealthDevUiProcessor.java

```diff
@@ -0,0 +1,30 @@
+package io.quarkus.smallrye.health.deployment;
+
+import io.quarkus.deployment.IsDevelopment;
+import io.quarkus.deployment.annotations.BuildStep;
+import io.quarkus.devui.spi.page.CardPageBuildItem;
+import io.quarkus.devui.spi.page.Page;
+
+/**
+ * This processor is responsible for the dev ui widget.
+ */
+public class SmallRyeHealthDevUiProcessor {
+
+    @BuildStep(onlyIf = IsDevelopment.class)
+    CardPageBuildItem create() {
+        CardPageBuildItem pageBuildItem = new CardPageBuildItem("Smallrye Health");
+
+        pageBuildItem.addPage(Page.externalPageBuilder("Health")
+                .icon("font-awesome-solid:heart-circle-bolt")
+                .url("/q/health")
```

Comment:
We should use `NonApplicationRootPathBuildItem`  here and generate the endpoint. See https://github.com/quarkusio/quarkus/blob/859c915c54b166ac397214f898209b7f84a7d17f/extensions/smallrye-openapi/deployment/src/main/java/io/quarkus/smallrye/openapi/deployment/devui/OpenApiDevUIProcessor.java#L19 as an example

Reply:
both approach will be broken once we merge the management interface :-D